pkg/services/apiserver/builder/helper.go

```diff
@@ -399,7 +399,8 @@ func InstallAPIs(
 			}); err != nil {
 				return err
 			}
-			if len(g.PrioritizedVersions) < 1 {
+
+			if len(g.PrioritizedVersions) == 0 {
 				continue
 			}
```

Comment:
Introduced by @toddtreece: it is a fix for supporting builders that don’t register stored kinds

Reply:
🤔 this doesn't make sense - given `len()` always returns a positive integer (or 0), `n < 1` and `n == 0` are identical.